### PR TITLE
Some feedback on models.py

### PR DESCRIPTION
**Pull request by gerwin3**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -23,7 +19,7 @@ def __init__(self, num_classes=15):
             torchvision.models.video.resnet.Conv2Plus1D(1, 32, 32, padding=2),
             torchvision.models.video.resnet.Conv2Plus1D(32, 16, 32, padding=1),
             torch.nn.MaxPool3d((3, 3, 3)),
-            Flatten(),
+            torch.nn.Flatten(startdim=1),
             torch.nn.Linear(in_features=576, out_features=256)
```
Comment (gerwin3) [inline models.py:23]: Zo'n linear laag hier kost je ontzettend veel weights, terwijl die misschien niet nodig is. Het is soms effectiever om nog een extra conv laag toe te voegen totdat de dimensionality laag genoeg is. Dan kan je de linear laag bewaren tot classificatie, waar die meestal het meest effectief is. Scheelt veel trainingstijd ook.

**Review by gerwin3 — COMMENTED** [review]
Wat betreft die Linear lagen, hetzelfde geld ook voor `self.classifier`. Evt. kan je die in een stap doen, dus van 512+256 naar `num_classes`. Maakt waarschijnlijk echter niet heel veel uit in de praktijk.

Als laatste, die `torch.cat([cnn_output, pose_cnn_output], dim=1)` is wel een zwak punt in het model, omdat je die semantisch gezien niet zomaar kan concatenaten (als ik het model goed begrijp). Je zou eigenlijk naar een two-stream model moeten gaan, dat is niet heel moeilijk.

In plaats van `concat` -> `classify`, doe je het andersom, dus iets van:

```
pred1 = self.classifier(pose_cnn_output)
pred2 = self.classifier(cnn_output)

pred = torch.mean(pred1, pred2)
```

Nu doe je pas fusion op de twee "stromen" wanneer de waardes ook semantisch equivalent zijn, nl., ze hebben beiden het doel tot correct classificeren van wie het is. Als je later de embedding eruit wil halen kan je hetzelfde doen op de feature-sets. (Soms werkt `torch.max` beter dan `torch.mean`, er zijn nog heel aantal andere fusion methodes zelfs, zie Google Scholar.)

File: models.py (hunk 1 above)
Comment (gerwin3) [reply models.py:23]: (Heb tevens ff die custom `Flatten` laag vervangen door de ingebouwde in `torch`)
